pkg/sdk/system_functions.go

```diff
@@ -7,6 +7,7 @@ import (
 
 type SystemFunctions interface {
 	GetTag(ctx context.Context, tagID ObjectIdentifier, objectID ObjectIdentifier, objectType ObjectType) (string, error)
+	GetTags(ctx context.Context, tagID ObjectIdentifier, objectID ObjectIdentifier, objectType ObjectType) ([]string, error)
```

Comment:
please add integration tests in `system_functions_integration_test.go`

Reply:
see below